src/main/java/HorizontalGenerator.java

```diff
@@ -0,0 +1,5 @@
+public class HorizontalGenerator {
```

Comment:
I like you create a whole class for printing a new line. Great for reusability and easy changes